src/backend/aspen/workflows/nextstrain_run/builds_templates/mega_template.yaml

```diff
@@ -73,6 +73,12 @@ subsampling:
           type: "proximity"
           focus: "group"
 
+    international_serial_sampling:
+      group_by: "region year month"
+      seq_per_group: 2
+      query: --query "(country != '{country}')"
+
+
```

Comment:
this is added so we still have some samples across all time to support a normal-looking tree. it's the same section in targeted tree